tests/test_date.py

```diff
@@ -82,6 +82,35 @@ def test_in(self):  # Same as "from now"
         self.assert_date('in 45 minutes', datetime(2017, 6, 16, 20, 22, 22))
         self.assert_date('in 45 seconds', datetime(2017, 6, 16, 19, 38, 7))
 
+    def test_since(self):
+        self.assert_date('since last may', datetime(2017, 5, 1))
+        self.assert_date('since last month', datetime(2017, 5, 1, 19, 37, 22))
+        self.assert_date('since last Friday', datetime(2017, 6, 9))
+        self.assert_date('since last Saturday', datetime(2017, 6, 10))
+        self.assert_date('since last Thursday', datetime(2017, 6, 15))
+        self.assert_date('since last week', datetime(2017, 6, 5))
+
+        self.assert_date('since Friday', datetime(2017, 6, 9))
+        self.assert_date('since Saturday', datetime(2017, 6, 10))
+        self.assert_date('since Thursday', datetime(2017, 6, 15))
+
+        self.assert_date('since 2016', datetime(2016, 1, 1))
+        self.assert_date('since July', datetime(2016, 7, 1))
+        self.assert_date('since April 2016', datetime(2016, 4, 1))
+        self.assert_date('since April 11, 2016', datetime(2016, 4, 11))
+
+        self.assert_date('since yesterday', datetime(2017, 6, 15))
+        self.assert_date('since this month', datetime(2017, 6, 1))
+        self.assert_date('since this week', datetime(2017, 6, 12))
```

Comment:
should be sunday the 11th => date time(2017,6,11)

Reply:
Changing the start of week is a big change. Will do a separate PR.